src/App.js

```diff
@@ -1,24 +1,99 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {useState} from 'react';
 import './App.css';
 
-function App() {
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+
+import FoodBox from './components/foodbox/FoodBox'
+import AddFood from './components/addfood/AddFood'
+import Search from './components/search/Search'
+
+const App = () => {
+
+  const [foodList, setFoodList] = useState(foods)
+  const [foodListClient, setFoodListClient] = useState(foods)
+  const [showForm, setShowForm] = useState(false)
+  const [todaysFoods, setTodaysFoods] = useState([])
+  const [totalTodaysCalories, setTotalTodaysCalories] = useState([])
+
+  const handleShowForm = () => {
+    setShowForm(true)
+  }
+
+  const handleNewFoodSubmit = (event) => {
+    event.preventDefault()
+
+    let newFoodName = event.target.name.value
+    let newFoodCalories = event.target.calories.value
+    let newFoodImg = event.target.image.value
```

Comment:
to reduce the code a bit you can directly create the obj with the values from the target instead of saving that into variables.